awscli/customizations/s3/utils.py

```diff
@@ -61,12 +71,34 @@ def human_readable_size(value):
     elif bytes_int < base:
         return '%d Bytes' % bytes_int
 
-    for i, suffix in enumerate(humanize_suffixes):
+    for i, suffix in enumerate(HUMANIZE_SUFFIXES):
         unit = base ** (i+2)
         if round((bytes_int / unit) * base) < base:
             return '%.1f %s' % ((base * bytes_int / unit), suffix)
 
 
+def human_readable_to_bytes(value):
+    """Converts a human readable size to bytes.
+
+    :param value: A string such as "10MB".  If a suffix is not included,
+        then the value is assumed to be an integer representing the size
+        in bytes.
+    :returns: The converted value in bytes as an integer
+
+    """
+    suffix = value[-2:].lower()
+    has_size_identifier = (
+        len(value) >= 2 and suffix in SIZE_SUFFIX)
+    if not has_size_identifier:
+        try:
+            return int(value)
+        except ValueError:
+            raise ValueError("Invalid size value: %s" % value)
```

Comment:
You might want to state what a valid size value qualifies as in the error message. I could see someone see Mib being used for ``ls --human-readable`` and try using that instead.